themes/10up-theme/tailwind.config.js

```diff
@@ -10,32 +34,60 @@ module.exports = {
 	 */
 	content: [
 		/* Ensure changes to all PHP, JS, and JSON files rebuild your CSS */
-		'./assets/js/**/*.js',
-		'./includes/**/*.{php,js,json}',
-		'./partials/**/*.php',
-		'./templates/**/*.php',
+		'404.php',
+		'archive.php',
+		'attachment.php',
+		'author.php',
+		'category.php',
+		'date.php',
+		'footer.php',
+		'front-page.php',
+		'functions.php',
+		'header.php',
+		'home.php',
+		'index.php',
+		'page.php',
+		'search.php',
+		'searchform.php',
+		'single.php',
+		'singular.php',
+		'tag.php',
+		'taxonomy.php',
+
+		// Directories
+		'assets/**/*.js',
+		'includes/**/*.{js, php}',
+		'partials/**/*.php',
+		'templates/**/*.php',
 	],
 	safelist: [
-		/* Prevent editor-specific styles from being purged */
-		'editor-post-title__block',
-		'editor-post-title__input',
-		'entry-content',
-		'entry-title',
-		'block-editor-block-list__layout',
+		{
+			pattern: /(bg|text)-(primary|secondary|tertiary)/,
+		},
```

Comment:
@iansvo I think we could add this as a default:
`pattern: /(is|has|js)-*/,`

Reply:
To confirm, are these classes that we would be authoring in our own CSS somewhere? If so, they won't need to be safelisted unless they're included in an `@layer` which we don't need to use here anyway (at least not until its officially supported in browsers).

Reply:
+1 on this just to be safe - no pun intended